statistics/update.go

```diff
@@ -92,6 +94,20 @@ func (s *SessionStatsCollector) StoreQueryFeedback(feedback interface{}) {
 	if q.histVersion == 0 || q.actual < 0 || !q.valid {
 		return
 	}
+
+	var rate float64
+	if q.actual == 0 {
+		if q.expected == 0 {
+			rate = 0
+		} else {
+			rate = 100
+		}
+	} else {
+		rate = math.Abs(float64(q.expected-q.actual)/float64(q.actual)) * 100
+		rate = math.Min(100, rate)
```

Comment:
I think 100 is too small, it's acceptable in many cases that actual is 2x larger than expected.

Reply:
But knowing that it is 2x larger than actual does not give more information than 1x larger?